platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationConfigInstaller.java

```diff
@@ -77,64 +72,56 @@ public ApplicationConfigInstaller(
       return;
     }
-    InputStream is = null;
+
+    LOGGER.trace("Found applications to install from config.");
+    for (Entry<Object, Object> curApp : props.entrySet()) {
+      String appName = (String) curApp.getKey();
+      String appLocation = (String) curApp.getValue();
+      LOGGER.debug("Starting app {} at location: {}", appName, appLocation);
+
+      startApp(appName, appLocation);
+    }
+    LOGGER.trace("Finished installing applications, uninstalling installer module...");
+
+    installAndUninstallFeatures();
+  }
+
+  private void startApp(String appName, String appLocation) {
     try {
-      is = new FileInputStream(configFile);
-      Properties props = new Properties();
-      props.load(is);
-      if (!props.isEmpty()) {
-        LOGGER.debug("Found applications to install from config.");
-        for (Entry<Object, Object> curApp : props.entrySet()) {
-          String appName = curApp.getKey().toString();
-          String appLocation = curApp.getValue().toString();
-          LOGGER.debug("Starting app {} at location: {}", appName, appLocation);
-
-          try {
-            if (StringUtils.isNotEmpty(appLocation)) {
-              appService.addApplication(new URI(appLocation));
-            }
-            executeAsSystem(
-                () -> {
-                  appService.startApplication(appName);
-                  return true;
-                });
-          } catch (ApplicationServiceException ase) {
-            LOGGER.warn("Could not start " + appName, ase);
-          } catch (URISyntaxException use) {
-            LOGGER.warn(
-                "Could not install application, location is not a valid URI " + appLocation, use);
-          }
-        }
-        LOGGER.debug("Finished installing applications, uninstalling installer module...");
-
-        try {
-          if (!StringUtils.isBlank(postInstallFeatureStart)) {
-            featuresService.installFeature(
-                postInstallFeatureStart, EnumSet.of(FeaturesService.Option.NoAutoRefreshBundles));
-          }
-          if (!StringUtils.isBlank(postInstallFeatureStop)) {
-            featuresService.uninstallFeature(postInstallFeatureStop);
-          }
-        } catch (Exception e) {
-          LOGGER.debug("Error while trying to run the post-install start and stop operations.", e);
-        }
-
-      } else {
-        LOGGER.debug("No applications were found in the configuration file.");
+      if (!appLocation.isEmpty()) {
+        appService.addApplication(new URI(appLocation));
+      }
+      executeAsSystem(
+          () -> {
+            appService.startApplication(appName);
+            return true;
+          });
+    } catch (ApplicationServiceException ase) {
+      LOGGER.warn("Could not start {}", appName, ase);
+    } catch (URISyntaxException use) {
+      LOGGER.warn(
+          "Could not install application, location is not a valid URI {}", appLocation, use);
```

Comment:
✏️  looks like a typo here `ase` instead of `use`

Reply:
I believe it was an acronym for the exception: `catch (ApplicationServiceException ase)`